src/traceViewer/web/ui/filmStrip.tsx

```diff
@@ -95,7 +95,7 @@ export const FilmStrip: React.FunctionComponent<{
     image.src = imageURL(context, previewVideo.fileName, previewIndex);
     await new Promise(f => image.onload = f);
     return image;
-  }, [previewVideo, previewMetaInfo, previewIndex, measure.width], undefined);
+  }, [previewMetaInfo, previewIndex, measure.width, context, previewVideo.fileName], undefined);
```

Comment:
This was not a bug, because `previewMetaInfo` depends on `context`, and `previewVideo` equality is stronger than `previewVideo.fileName`. However, linter is nice :)